evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -104,6 +104,9 @@ def _create_ensemble(self):
         problem_type = best_pipelines[0]["pipeline"].problem_type
         n_jobs_ensemble = 1 if self.text_in_ensembling else self.n_jobs
         input_pipelines = []
+        cached_data = {
+            mf: x["cached_data"] for mf, x in self._best_pipeline_info.items()
```

Comment:
What are mf and x here?

Reply:
mf is model family and x is the value associated. The `_best_pipeline_info` dictionary is structured as 
```python
_bpi = {ModelFamily.XGBOOST: {"pipeline": ..., other_info_here}}

```